pkg/login/interactive_login.go

```diff
@@ -46,6 +56,52 @@ func InteractiveLogin(config *config.Config) error {
 	return nil
 }
 
+// DisplayName returns the display name for a successfully authenticated user
+func DisplayName(account *Account, baseURL string, apiKey string) (string, error) {
```

Comment:
re: next comment about `getAccount`, this should probably be moved to the same file.